pkgs/tools/networking/proxychains-ng/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "proxychains-ng";
+  version = "4.15";
+
+  src = fetchFromGitHub {
+    owner = "rofl0r";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "128d502y8pn7q2ls6glx9bvibwzfh321sah5r5li6b6iywh2zqlc";
+  };
+
+  meta = with lib; {
+    description = "A preloader which hooks calls to sockets in dynamically linked programs and redirects it through one or more socks/http proxies";
+    homepage = "https://github.com/rofl0r/proxychains-ng";
+    license = licenses.gpl2Plus;
+    maintainers = with maintainers; [ zenithal ];
+    platforms = platforms.linux;
```

Comment:
Is it possible to change this to `platforms.unix`?

Reply:
According to the README of https://github.com/rofl0r/proxychains-ng, it is a unix program and it supports Linux, BSD, Mac, Haiku.

I would like to make this change, but I could not test it.

Reply:
Maybe just check results from ofborg.

Reply:
All checks have passed

Reply:
[It is falling on darwin](https://logs.nix.ci/?key=nixos/nixpkgs.137541&attempt_id=84056fd3-fec8-4c13-98a0-84eaa683459f) (sorry we are lack of darwin reviewers and CI checks for darwin take long time).

Reply:
Then I revert back to `platforms.linux`